tools/dgeni/templates/constant.template.html

```diff
@@ -6,9 +6,7 @@ <h4 id="{$ constant.name $}" class="docs-header-link docs-api-h4 docs-api-consta
 </h4>
 
 {%- if constant.isDeprecated -%}
-<div class="docs-api-constant-deprecated-marker" {$ macros.deprecationTitle(constant) $}>
-  Deprecated
```

Comment:
Should we always keep `Deprecated` for accessibility reasons?

Reply:
I don't fully remember how it's showing up right now, but I was thinking we want some static text, next to the "class name" showing "Deprecated". That one you can hover for more details?

Reply:
Right now we do show static text ("Deprecated") text next to class name but in this commit I removed it that next to class name we dont show hard coded "Deprecated" text anymore but instead the class name self is styled to looked like deprecated and upon hovering it we show the deprecated message.

[heres a screenie I attached at upstream doc site](https://github.com/angular/material.angular.io/pull/1274#issuecomment-2409422779). lemme know if this is what you mean.

Reply:
I reverted that change and kept the `Deprecated` text instead of adding it directly from the component on docs site.